scripts/templates/stype_map_helpers.hpp.mako

```diff
@@ -7,7 +8,7 @@ from templates import helper as th
     x=tags['$x']
     X=x.upper()
 %>
-// This file is autogenerated from the template at ${self.template.filename}
+// This file is autogenerated from the template at ${os.path.dirname(self.template.filename)}/${os.path.basename(self.template.filename)}
```

Comment:
is this related?

Reply:
yeah, generated file contains a comment:
```
// This file is autogenerated from the template at templates/stype_map_helpers.hpp.mako
```

on Windows, there's different separator (-> `\`) and it's displayed as a diff in the generated files (causing failure)

Reply:
alternatively we could just drop this comment 😉

Reply:
ok, lets leave it. thanks.